eng/templates/Azure.ResourceManager.Template/src/Azure.ResourceManager.Template.csproj

```diff
@@ -5,4 +5,9 @@
     <Description>Azure management client SDK for Azure resource provider ProviderFullName</Description>
     <PackageTags>azure;management;ProviderNameLowercase</PackageTags>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- TODO: Temp workaround. Should be fixed prior to release  -->
```

Comment:
Should this be in the template? Is there an issue tracking the fixing of the problems? If so please add a link here.

Reply:
The AZC0001 is a tricky one as enabling the approved RP namespace can only be done after the initial check-in and successful build. We can only use release process to change afterward.  For AZC0008 should be disabled at infra level.